ext/pcre/php_pcre.c

```diff
@@ -3001,6 +3111,10 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_preg_grep, 0, 0, 2)
     ZEND_ARG_INFO(0, flags)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO_EX(arginfo_preg_callout_function, 0, 0, 0)
+	ZEND_ARG_INFO(0, new_function)
```

Comment:
What do you think to type the `$new_function` argument, with `ZEND_ARG_CALLABLE_INFO` for instance?

Reply:
I'm holding off on adding arginfo types until we can start adding them wholesale, to avoid behavior discrepancies. https://wiki.php.net/rfc/consistent_type_errors was the largest piece of preparation for that, but there's some more work necessary before we can start doing that.

Reply:
:+1: